Lesson4HomeWork.playground/Contents.swift

```diff
@@ -0,0 +1,21 @@
+import UIKit
+
+var arrayNumbers: [Int] = [-1, 2, 6, 12, 0, 5, 7, 8]
+
+for elements in 0...arrayNumbers.count - 1 {
```

Comment:
There's nothing wrong with closed range like this (0... arrayNumbers.count - 1) but you can optimise this line using simple range 0..< arrayNumbers.count